### PR TITLE
Truncate team name

### PR DESCRIPTION
**Pull request by alkeshrails**
Truncate Team name

### CODE REVIEW

File: app/views/challenges/show/_top_five_leaderboards.html.erb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<% truncate = truncate.present? ? truncate : false %>
```
Comment (piotrekpasciak) [inline app/views/challenges/show/_top_five_leaderboards.html.erb:1]: I think you mean more something like `truncate.defined?`.

File: app/views/leaderboards/_leaderboards.html.erb [hunk 2]
```diff
@@ -2,6 +2,7 @@
   <%
     participants = leaderboard_participants(leaderboard)
     challenge    = leaderboard.challenge
+    truncate     = truncate.present? ? truncate : false
```
Comment (piotrekpasciak) [inline app/views/leaderboards/_leaderboards.html.erb:5]: `truncate.defined?`

Comment (alkeshrails) [reply app/views/leaderboards/_leaderboards.html.erb:5]: undefined method `defined?' for "true":String

File: app/views/challenges/show/_top_five_leaderboards.html.erb (hunk 1 above)
Comment (alkeshrails) [reply app/views/challenges/show/_top_five_leaderboards.html.erb:1]: I get error undefine method ``defined?`` for nil or string.

**Comment by alkeshrails** [comment]
undefined method `defined?' for "true":String

**Pull request merged**